pkg/kubelet/cm/cgroupManager_mock.go

```diff
@@ -0,0 +1,45 @@
+// +build linux
+
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cm
+
+import (
+	"github.com/stretchr/testify/mock"
+)
+
+// MockCgroupManager is a mock object which implements the cm.CgroupManager interface
+type MockCgroupManager struct {
```

Comment:
i guess... not clear where this is being aggregated from, i suspect in kubelet_test.go somewhere eventually.  seems like the mock could be local to there as well.  not a huge deal.  tbh, the mocks are making it hard to follow the PR some since they are not actually used yet.